app/src/main/java/org/fossasia/pslab/activity/MainActivity.java

```diff
@@ -76,6 +83,10 @@
     private Handler mHandler;
     private ScienceLabCommon mScienceLabCommon;
 
+    public boolean PSLabisConnected = false;
+
+    InitializationVariable initialisationStatus;
```

Comment:
Can this be private?

Reply:
Hi @Vikum94 ! It is used in another fragment in the same package. So I left it in the package scope!